client/ayon_deadline/abstract_submit_deadline.py

```diff
@@ -169,6 +176,7 @@ def _append_job_output_paths(self, instance, job_info):
             due to remapping workfile to published workfile.
         Used in JobOutput > Explore output
         """
+        self.log.info(f"paths::{instance.data['expectedFiles']}")
```

Comment:
```suggestion
        self.log.debug(f"paths::{instance.data['expectedFiles']}")
```

Reply:
Preferably the logged message is clearer to so that it's clear _what_ it logged. Like "Instance expected files: {}".